### ISSUE TITLE
[A11Y] Control not visible in High Contrast

### ISSUE THREAD

**Issue by PhilibertDugas** [paypal/paypal-checkout#786]
The Close control is implemented with a CSS `background-color` properties which do not appear in Windows High Contrast themes. This makes it difficult to return to the previous screen. Low-vision, high-contrast users should have equal access to content and imagery that sighted users have access to.

**HTML:**
```html
<a href="#" class="paypal-checkout-close"></a>
```
**CSS:**
```css
#xcomponent-paypal-checkout-14789aa82a.paypal-checkout-background-color-black .paypal-checkout-close:before, #xcomponent-paypal-checkout-14789aa82a.paypal-checkout-background-color-black .paypal-checkout-close:after {
    background-color: #fff;
}
```
## Steps to reproduce

1. Go to https://www.chubbiesshorts.com/ with Internet Explorer
2. Add some items to the cart
3. Proceed to the Cart
4. Activate the PayPal control
5. `Cmd` + `~` back to the main window
6. With an active High Contrast theme, review the Close control

### Results

**Expected:**
- Borders to be visible in High Contrast theme.

**Actual:**
- Borders are missing in High Contrast theme.

## Recommendation

- Use the CSS `border` property to generate a clearly visible Close control in High Contrast.

**CSS:**
```css
#xcomponent-paypal-checkout-14789aa82a.paypal-checkout-background-color-black .paypal-checkout-close:before, #xcomponent-paypal-checkout-14789aa82a.paypal-checkout-background-color-black .paypal-checkout-close:after {
    border: solid #fff 1px;
}
```
## Additional information

- **WCAG Principle:** Perceivable
- **WCAG SC:** 1.4.1 Use of Color
- **Severity:** Medium

## Screenshots
<img width="702" alt="screen shot 2018-03-05 at 2 22 31 pm" src="https://user-images.githubusercontent.com/1392632/36998429-e7dd334a-208a-11e8-9af9-4ff70103c36e.png">
<img width="702" alt="screen shot 2018-03-05 at 2 22 31 pm" src="https://user-images.githubusercontent.com/1392632/36998486-178d6e5c-208b-11e8-9c18-65732e68b011.png">

**Comment by bluepnume** [comment]
@mcntsh can you take this since you're already working on improving the contrast here?

**Comment by mcntsh** [comment]
Addressed in PR #898 and will be out in the next release!
